.drone/drone.yml

```diff
@@ -132,6 +132,7 @@ trigger:
   - pull_request
 type: docker
 ---
+# TODO: Remove this pipeline once its GitHub actions equivalent is battle tested.
```

Comment:
```suggestion
```

We will need to remove this line. If we edit drone.yml without running `make drone` to generate it from the jsonnet, Drone will refuse to run the pipelines automatically. We'd have to manually click a "run" button each time a PR is opened (see #2557).